azure_functions_worker/testutils_lc.py

```diff
@@ -0,0 +1,268 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+from typing import Dict
+
+import base64
+import json
+import os
+import re
+import subprocess
+import sys
+import time
+import uuid
+
+from Crypto.Cipher import AES
+from Crypto.Hash.SHA256 import SHA256Hash
+from Crypto.Util.Padding import pad
+import requests
+
+# Linux Consumption Testing Configs
+DOCKER_PATH = "DOCKER_PATH"
+DOCKER_DEFAULT_PATH = "docker"
+MESH_IMAGE_URL = "https://mcr.microsoft.com/v2/azure-functions/mesh/tags/list"
+MESH_IMAGE_REPO = "mcr.microsoft.com/azure-functions/mesh"
+DUMMY_CONT_KEY = "MDEyMzQ1Njc4OUFCQ0RFRjAxMjM0NTY3ODlBQkNERUY="
```

Comment:
Things which are doing to be constant and not changing, add an '_' to then as prefix.